code/game/gamemodes/gang/recaller.dm

```diff
@@ -280,10 +280,13 @@
 						usr << "<span class='warning'>There's not enough room here!</span>"
 						return
 
-				if(gang.points >= 30)
-					item_type = /obj/machinery/dominator
-					usr << "<span class='notice'>The <b>dominator</b> will secure your gang's dominance over the station. Turn it on when you are ready to defend it.</span>"
-					pointcost = 30
+				if(usrarea.type in gang.territory | gang.territory_new)
```

Comment:
```|``` on two lists produces a list containing the contents of both lists with no duplicates.
It's typically written without spaces though @Razharas so please remove the spaces.